beeswax/logs/ad_log.proto

```diff
@@ -212,7 +214,7 @@ message ConversionLogMessage {
   optional bool is_gdpr = 10 [default = false];
 }
 
-// Next Tag: 7
+// Next Tag: 8
```

Comment:
This tag counter should only be updated when we add a new field to the AdLogMessage, so it should probably stay at 7

Reply:
It was updated to 8 in our repo, so I updated here to reflect, but I don't see a 7 there.

Reply:
This seems to have been updated by Gil to include a field related to vast errors, but the field itself doesn't seem to have been added.